src/api/IFXAPI.js

```diff
@@ -633,7 +633,50 @@ export default class IFXAPIService {
 
   get account() {
     const baseURL = this.urls.ACCOUNTS
-    return this.genericAPI(baseURL, Account)
+    const createFunc = (accountData, decompose = false) => {
+      const newAccountData = cloneDeep(accountData)
+      // Initialize contacts and users as empty arrays - will be filled in if incoming orgData has contacts or users
+      newAccountData.user_accounts = []
+      newAccountData.user_product_accounts = []
+
+      // Check if incoming accountData has user_accounts, user_product_accounts
+      if (accountData.user_accounts && accountData.user_accounts.length) {
```

Comment:
Do you want to use the `?.` operator here or are there cases where that won't work?

```suggestion
      if (accountData.user_accounts?.length) {
```

Reply:
Done